db/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "camelcase": ["error", {properties: "never"}]
+  }
+}
```

Comment:
My guts tell me that we should add this as an override in the base .eslint config file (similar to other rules specific to folders).